README.md

```diff
@@ -256,6 +256,29 @@ Book.tracked_embeds_many          #=> ["pages"]
 Book.tracked_embeds_many?(:pages) #=> true
 ```
 
+**Skip soft-deleted embedded objects with nested tracking**
```

Comment:
Maybe the title is misleading and this is about the paranoia field, not about embedded objects tracking.